config/config-example.js

```diff
@@ -65,8 +65,8 @@ const fs = require('fs');
 exports.ssl = {
 	port: 443,
 	options: {
```

Comment:
Also, why do we need an `options` table anyway? We can just have `port`, `key`, `cert` at the top level.

Reply:
I'm not sure why I made an options table

Reply:
Oh, the options object is there to get passed to `https.createServer` as is